src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -4108,7 +4105,7 @@ void func_80831454(Player* this) {
     }
 }
 
-s32 Player_SetAction(PlayState* play, Player* this, PlayerActionFunc actionFunc, s32 arg3) {
+s32 Player_SetupAction(PlayState* play, Player* this, PlayerActionFunc actionFunc, s32 arg3) {
```

Comment:
I think the reason to not name this function `SetupAction` like everywhere else was because it does more than what a standard `SetupAction` function does and I originally wanted to distinguish it somehow. Maybe just a plain comment explaining this would be enough tho

Reply:
Ah, I see. Yeah it's definitely not a wrapper for just applying the `Action`, but then ignoring the typical convention of `_SetupAction`, in isolation I would think `SetAction` would be the wrapper where all you do is set the action, and `SetupAction` would be the one where you do `SetAction` plus more

Reply:
Yeah, in isolation they would be named the other way around, but we are kinda stuck with this :cry:

Reply:
I'd be interested in exploring this more, but a change like that would need every actor decompiled first due to the scope of it. I can keep it `Player_SetAction` and `Player_SetUpperAction` then